packages/components/src/molecules/Rating/Rating.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { forwardRef, useState } from 'react'
+import type { ReactNode } from 'react'
+import type { HTMLAttributes } from 'react'
+
+import List from '../../atoms/List'
+import { Icon, IconButton } from '../..'
+
+export interface RatingProps extends HTMLAttributes<HTMLUListElement> {
+  /**
+   * The current value of the rating, based on the quantity of child elements.
+   */
+  value: number
+  /**
+   * A React component that will be rendered as an icon.
+   */
+  icon: ReactNode
+  /**
+   * Specifies that this input should be controled by the user.
+   */
+  actionable?: boolean
+  /**
+   * ID to find this component in testing tools (e.g.: cypress, testing library, and jest).
+   */
+  testId?: string
+}
+
+export interface RatingItemProps {
+  'data-fs-rating-item'?: 'full' | 'partial' | 'empty'
+  'data-testid'?: string
+}
+
+const Rating = forwardRef<HTMLUListElement, RatingProps>(
+  function Rating(
+    { children, testId = 'fs-rating', value, actionable, icon, ...otherProps },
+    ref
+  ) {
+    const [rating, setRating] = useState(value - 1)
+    const [hover, setHover] = useState(0)
```

Comment:
Need help to fix the hover of first star 🥺 

https://user-images.githubusercontent.com/11613011/211648534-60b6bee9-df9c-42d1-8a64-98efdefac664.mov

Reply:
I checked it again, and it seems to be fixed now :)